.github/workflows/CI.yml

```diff
@@ -130,6 +130,9 @@ jobs:
           sudo cp ./tests/harbor_ca.crt /usr/local/share/ca-certificates/
           sudo update-ca-certificates
           sudo service docker restart
+          wget https://get.helm.sh/helm-v3.0.0-linux-386.tar.gz && tar zxvf helm-v3.0.0-linux-386.tar.gz && \
```

Comment:
use 3.1 instead?

Reply:
Yes, helm 3.1.1 has replaced V3.0.0 in CI environment.
 
"Saving to: ‘helm-v3.1.1-linux-386.tar.gz’"